bazel/envoy_test.bzl

```diff
@@ -138,6 +139,30 @@ def envoy_cc_fuzz_test(
         tags = ["fuzz_target"] + tags,
     )
 
+    fuzzing_corpus(
+        name = name + "_corpus_dir",
+        srcs = [corpus_name],
+        testonly = True,
+    )
+
+    # Target for regression test
+    fuzzing_launcher(
+        name = name,
+        target = name + "_binary",
+        corpus = name + "_corpus_dir" if corpus_name else None,
+        is_regression = True,
```

Comment:
can the two targets be consolidated with a select for `"@envoy//bazel:libfuzzer"` choosing `False` or `True` for default?

Reply:
Just to clarify, do you mean both targets are merged into one target, say `name`, then passes the `is_regression` using `select`?

Reply:
if possible yes! that way developer invokes the targets the same as before